modules/fiftyonedegrees/devicedetection/no_cgo.go

```diff
@@ -0,0 +1,13 @@
+//go:build !cgo
+
+package devicedetection
+
+import (
+	"encoding/json"
+
+	"github.com/prebid/prebid-server/v3/modules/moduledeps"
+)
+
+func Builder(rawConfig json.RawMessage, _ moduledeps.ModuleDeps) (interface{}, error) {
+	panic("Not implemented when CGO_ENABLED=0")
```

Comment:
Please replace panic() with `return nil, errors.New("Not implemented when CGO_ENABLED=0")`
Panic isn't a good practice in apps.

Reply:
perhaps in this particular case panic is a good choice - it will tell the dev immediately that they need to `enable:false` the module in their config or enable `cgo`.  as clearly user either unintentionally compiled without cgo and forgot to disable the module, or accidentally disabled cgo - so better to signal right away.  the error might go unnoticed for some time.

Reply:
If the code cannot be used without CGO don't let to compile it.
The good to put a build tag `//go:build cgo` and compiler excludes this module from the output binary and the dev even able to use the module

Reply:
At the moment, https://github.com/prebid/prebid-server/blob/master/modules/builder.go#L13 modules/adapaters/etc are all enabled in one place - until/unless we make a mechanism to turn prebid into a library or figure out some great dynamic module (which I'm less of a fan of) mechanism, the Builder method must exist. All of the code that existed before has the build flag in the "must have cgo" and this file has the "when we don't have cgo" in order to allow that method to exist but, when used, panic (or error) as its only course of action.

Another idea that would solve this, would be to reveal cgo as a variable (also using build tags) and conditionally don't even configure that.  I'll try that approach…

Reply:
DONE